bears/jinja2/Jinja2Bear.py

```diff
@@ -184,6 +186,45 @@ def check_for_variable_spacing_issues(self,
                     end_column=m.end(0) + 1,
                     diffs=diff)
 
+    def check_for_statement_spacing_issues(self,
+                                           file,
+                                           filename,
+                                           line,
+                                           line_number,
+                                           statement_spacing):
```

Comment:
You have to give default value here right? and datatype isnt defined as well ;)

Reply:
Default values and their data type has been defined in the run function already.